### DISCUSSION TITLE
Photoprism does not fully view images and videos in the directory.

### DISCUSSION THREAD

**Discussion by pilgrimage80** [photoprism/photoprism#4683] [Q&A]
I have 55 thousand photos in my directory but when I scan it it sees 36 thousand of them, what could be the reason for this? How can I see what it skips in the directory.

**Comment by omerdduran** [comment]
Hi. You can find our troubleshooting checklist for missing pictures here:

https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

Library > Errors section will show you specifically which files had issues. For seeing the detailed process happening in real-time while files are being indexed, use the Library > Logs section

**Reply by omerdduran** [reply]
You can also check out the following question in the FAQ for more information on this topic: "Why does the count in Search not match the count of files in Originals?"

https://docs.photoprism.app/user-guide/faq/#media-library
